### DISCUSSION TITLE
S Pen support

### DISCUSSION THREAD

**Discussion by renatogodois** [Adventech/sabbath-school-android#659] [Ideas]
Hi team, Maranatha!

I don't know if you have this kept in mind but I would like to suggest for you, add S Pen support to use with Samsung devices. I would be a great help.

Thank you.

**Comment by VitalikL** [comment]
Thanks @renatogodois . Sorry for the late reply. I am not sure if this is still relevant, but I would love to hear more specifics about the S Pen support.
